internal/delegatedauth/delegatedauth.go

```
@@ -0,0 +1,297 @@
+	if err != nil {
+		return bundle, false, err
+	}
+	req.Header.Set("Content-Type", "application/json")
+	resp, err := client.Do(req)
+	if err != nil {
+		if errors.Is(reqCtx.Err(), context.DeadlineExceeded) {
+			return bundle, false, fmt.Errorf("%w: refresh timed out", ErrRefreshRejected)
+		}
+		return bundle, false, err
+	}
+	defer resp.Body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `resp.Body.Close` is not checked

(errcheck)